Source/Core/ArcGisImageServerTerrainProvider.js

```diff
@@ -231,7 +237,17 @@ define([
             url = proxy.getURL(url);
         }
 
-        var promise = RequestScheduler.throttleRequest(url, loadImage);
+        if (!defined(request)) {
+            // If a request object isn't provided, perform an immediate request
+            request = new Request();
```

Comment:
The only reason I didn't do this in a few places was to avoid the memory allocation, but maybe it doesn't matter.